wpilib/wpilib/command/subsystem.py

```diff
@@ -138,36 +133,36 @@ def getCurrentCommand(self):
         """
         return self.currentCommand
 
-    def __str__(self):
-        return self.getName()
+    def getCurrentCommandName(self):
+        """
+        Returns the current command name, or empty string if no current command.
 
-    def getName(self):
-        """Returns the name of this subsystem, which is by default the class
-        name.
-        
-        :returns: the name of this subsystem
+        :returns: the current command name
+        """
+        currentCommand = self.getCurrentCommand()
+        if currentCommand is not None:
+            return currentCommand.getName()
+        return ""
+
+    def addChild(self, child, name=None):
         """
-        return self.name
-
-    def getSmartDashboardType(self):
-        return "Subsystem"
-
-    def initTable(self, table):
-        super().initTable(table)
-        if table is not None:
-            self.hasDefaultEntry = table.getEntry("hasDefault")
-            self.defaultEntry = table.getEntry("default")
-            self.hasCommandEntry = table.getEntry("hasCommand")
-            self.commandEntry = table.getEntry("command")
-
-            if self.defaultCommand is not None:
-                self.hasDefaultEntry.setBoolean(True)
-                self.defaultEntry.setString(self.defaultCommand.getName())
-            else:
-                self.hasDefaultEntry.setBoolean(False)
-
-            if self.currentCommand is not None:
-                self.hasCommandEntry.setBoolean(True)
-                self.commandEntry.setString(self.currentCommand.getName())
-            else:
-                self.hasCommandEntry.setBoolean(False)
+        Associate a :class:`.Sendable` with this Subsystem.
+        Update the child's name if provided
+        :param child: sendable
```

Comment:
Add a blank line before this.

Reply:
And capitalize Sendable

Reply:
@Twinters007 File a bug upstream if you care.

Reply:
We can differ on capitalization, no?

Reply:
I don't see any reason to.

Reply:
Looks better in the docs imo.

Reply:
Inconsistency is bad, and _doesn't_ look good.

Reply:
I think change it to be Right here _and_ send a PR upstream.

Reply:
We should also change the rest of them to be consistent then.